pallets/cash/src/chains.rs

```diff
@@ -1004,18 +1265,28 @@ impl Chain for Polkadot {
     fn hash_string(_hash: &Self::Hash) -> String {
         panic!("XXX not implemented");
     }
+
+    fn chain_account(address: Self::Address) -> ChainAccount {
+        ChainAccount::Dot(address)
+    }
+
+    fn chain_block(_block: Self::Block) -> ChainBlock {
+        panic!("XXX not implemented");
+    }
 }
 
 pub fn get_chain_account(chain: String, recipient: [u8; 32]) -> Result<ChainAccount, Reason> {
-    match &chain.to_ascii_uppercase()[..] {
-        "ETH" => {
-            let mut eth_recipient: [u8; 20] = [0; 20];
-            eth_recipient[..].clone_from_slice(&recipient[0..20]);
+    let chain_account_fn = match &chain.to_ascii_uppercase()[..] {
+        "ETH" => ChainAccount::Eth,
+        "MATIC" => ChainAccount::Matic,
+        "MAT" => ChainAccount::Matic,
```

Comment:
Curious about the either here.

Reply:
this is the whole `bytes4` thing :( i use `MAT:` for the magic header and MATIC for addresses etc. Do you want me to rename the other ones to `MAT` just seemed odd since `eth` is the ticker for Ethereum and `MATIC` is the ticker for Polygon

We can really name these whatever we want but i think it might be nice to use the tickers since they're already defacto market imposed short names on each chain we might bridge